docs/DeveloperGuide.adoc

```diff
@@ -297,33 +298,33 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new task | check the details of the task when I want
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |edit an existing task |update task information when I need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |tutor |check the time I spent on preparing tutorial materials |ask for salary
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |forgetful tutor |be reminded before my tutorials |go for the tutorials on time
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`*` |caring tutor |check the upcoming events |remind my students
```

Comment:
Events: Does event refer to class schedules or to things like deadlines (ie are regularly scheduled or one-off things)?